### ISSUE TITLE
Add Auction4Reputation.getCurrentAuction

### ISSUE THREAD

**Issue by dkent600** [daostack/arc#548]
Would be good to add a function `Auction4Reputation.getCurrentAuction` that looks like the following:

```
function currentAuction() public view returns(uint)
{
  return (now - auctionsStartTime)/auctionPeriod;
}

**Comment by dkent600** [comment]
This isn't the reason for requesting this, but `bid` should use the function, to avoid duplicating logic.

**Comment by orenyodfat** [comment]
This can be calculated off chain

**Comment by dkent600** [comment]
Yes, it can be calculated off chain, but requiring that of users has two bad implications:

1) it requires that users discover the inner logic of the contract by digging into the public Arc documentation, finding nothing there, then digging into the Arc solidity source code (remember, not everyone is using arc.js, by a long shot, not everyone would know where to look, and not everyone is comfortable with solidity).  Digging into source code is not how Arc should have to be used.

2) Even if this logic were documented in the public Arc documentation, duplication of the inner logic outside of the contract means that if ever the logic changes, eveyone who has duplicated that logic will suddenly find their code broken and requiring change.

This is a super easy change.  Why not do it?  The additional cost of deploying the contract is trivial.
